code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -89,6 +89,11 @@
 
 	var/shouldwakeup = FALSE //convenience var for forcibly waking up an idling AI on next check.
 
+	//Audio
+	var/speak_audio = "" //the sound that will play when the animal speaks
```

Comment:
these should be null instead of empty strings

Reply:
put an audio clip of you saying "uh oh" as the default sound